Doc/library/heapq.rst

```diff
@@ -110,17 +110,17 @@ The module also offers three general purpose functions based on heaps.
 
    Return a list with the *n* largest elements from the dataset defined by
    *iterable*.  *key*, if provided, specifies a function of one argument that is
-   used to extract a comparison key from each element in the iterable:
-   ``key=str.lower`` Equivalent to:  ``sorted(iterable, key=key,
-   reverse=True)[:n]``
+   used to extract a comparison key from each element in the iterable (for example,
```

Comment:
I think it would be more clear for this and the following reference to be `*iterable*` like you did in the `Doc/library/functions.rst` part of this change.

Reply:
agreed and fixed.